ui/src/components/WorkerManagerWorkerTypesTable/index.jsx

```diff
@@ -0,0 +1,201 @@
+          </TableCellListItem>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.runningCapacity}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{workerType.pendingTasks}</Typography>
+        </TableCell>
+
+        <CopyToClipboard
+          title={`${workerType.lastActive} (Copy)`}
+          text={workerType.lastActive}>
+          <TableCell>
+            <TableCellListItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={workerType.lastActive} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <CopyToClipboard
+          title={`${workerType.lastResolved} (Copy)`}
+          text={workerType.lastResolved}>
+          <TableCell>
+            <TableCellListItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={workerType.lastResolved} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <TableCell>
+          <Typography>{failedNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{exceptionNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <Typography>{unscheduledNumber}</Typography>
+        </TableCell>
+
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/providers/${workerType.provider}`}>
+            <ListItemText
+              disableTypography
+              primary={<Typography>{workerType.provider}</Typography>}
+            />
+            <LinkIcon size={iconSize} />
+          </TableCellListItem>
+        </TableCell>
+      </TableRow>
+    );
+  };
+
+  render() {
+    const { workerTypes, searchTerm } = this.props;
+    const { sortBy, sortDirection } = this.state;
+    const sortedWorkerTypes = this.sortWorkerTypes(
+      workerTypes,
+      sortBy,
+      sortDirection,
+      searchTerm
+    );
+
+    return (
+      <DataTable
+        items={sortedWorkerTypes}
+        headers={[
+          'Worker Type',
+          'Pending Tasks',
+          'Running Capacity',
+          'Pending Capacity',
+          'Last Active',
+          'Last Resolved',
+          'Failed',
+          'Exception',
+          'Unscheduled',
+          'Provider',
+        ]}
+        sortByHeader={sortBy}
+        sortDirection={sortDirection}
```

Comment:
I think sorting  is not working for the failed, exception, and unscheduled columns.

![Screen Shot 2019-04-23 at 3 47 09 PM](https://user-images.githubusercontent.com/3766511/56611443-ce9cb680-65df-11e9-8cbe-520e6cb5daa1.png)

Reply:
done